x-pack/filebeat/input/cel/input.go

```diff
@@ -4,7 +4,7 @@
 
 // Package cel implements an input that uses the Common Expression Language to
 // perform requests and do endpoint processing of events. The cel package exposes
-// the github.com/elastic/mito/lib CEL extension library.
+// the https://github.com/elastic/mito/tree/dev/lib CEL extension library.
```

Comment:
No, this is not a link, it's a package import path.

It could be simplified™ making use of gopls by rewriting this as `[lib]`, but it doesn't add anything more than what is there now. We could do
```
// the [mito/lib] CEL extension library.
//
// [mito/lib]: https://pkg.go.dev/github.com/elastic/mito/lib
```
but that depends on godoc (rendered by pkg.go.dev) and pkg.go.dev will not display beats correctly™ (or at all for x-pack due to licenses).

So I think we are OK leaving this as is.